scripts/memcache-stats.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 argument="$1"
+
+IP_ADDR=$(hostname -I | sed 's/ /\n/g' | grep '10\.')
+CLEAN_IP_ADDR=$(echo "$IP_ADDR" | sed 's/\./-/g')
```

Comment:
Simpler: 
```
▶  hostname -I | grep -oE '^\S+' | sed 's/\./-/g'
10-0-0-239
```

Reply:
The man page for this explicitly says:

> Do not make any assumptions about the order of the output.

Reply:
Thanks for catching that!

```
 hostname -I | grep -oP "\b10\.\d+\.\d+\.\d+" | sed 's/\./-/g'
```

Reply:
There are a lot of fun way to do this...

```
hostname -I | tr ' ' '\n' | grep -E '^10\.'
```

Reply:
```
curl http://169.254.169.254/latest/meta-l-ipv4 | tr '.' '-'

Reply:
How about a shortened version of yours @danielbeardsley ?
```
hostname -I | grep -oP "\b10[.\d]+"
```

Reply:
One more :-)

```
hostname -I | grep -oP "\b10(\.\d]+){3}"
```
Yours would match `101.` and `1012345` ...

Reply:
Oooh... i like that :laughing:

Reply:
Wait, that doesn't work though.
```
hostname -I | grep -oP "\b10\.[.\d]+"
```
How about that?

Reply:
```
▶  echo "10.100000000" | grep -oP "\b10\.[.\d]+"
10.100000000
```

I accidentally left a bracket in mine, works after removal:
```
▶  hostname -I | grep -oP "\b10(\.\d+){3}"
10.0.0.239
```